### ISSUE TITLE
Disabled but still running

### ISSUE THREAD

**Issue by dbashyal** [AOEpeople/Aoe_Scheduler#89]
I disabled "enterprise_refresh_index" but it;s still running.

**Comment by dbashyal** [comment]
found and used this line to run cron instead and it excluded indexing. But still would love to exclude 'always' using AOE.
`
*/5 * * * *  /usr/bin/env SCHEDULER_BLACKLIST='enterprise_refresh_index' /bin/sh /.../cron.sh cron.php -m=default
`

**Comment by tegansnyder** [comment]
@danielpoe - you sparked my interest. What's your use case for disabling `enterprise_refresh_index` ? Are you running your own nightly reindex?

**Comment by dbashyal** [comment]
Yes we trigger reindex only when we need to. we don't use magento to update products from cms. everything is automated using API as we have different system to update products / inventory ...

**Comment by fbrnc** [comment]
Hi @dbashyal,
please try Aoe_Scheduler v1 (and update your cron group configuration) and let me know if you can reproduce the problem.
